### PR TITLE
Create DetectFloatingPointNumbers.py

### PR DESCRIPTION
**Pull request by HighnessAtharva**
Missing problem added with working solution.

### CODE REVIEW

File: RegexandParsing/DetectFloatingPointNumbers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Subdomain : Regex and Parsing
+Domain    : Python
+Author    : Atharva Shah
+Created   : 3 April 2021
+Problem   : https://www.hackerrank.com/challenges/introduction-to-regex/problem
+'''
+
+from re import match, compile
+
+pattern = compile('^[-+]?[0-9]*\.[0-9]+$')
```
Comment (arsho) [inline RegexandParsing/DetectFloatingPointNumbers.py:12]: This regex pattern can be changed to `compile('^[-+]?\d*\.\d+$')`

**Review by arsho — CHANGES_REQUESTED** [review]
@HighnessAtharva 

This solution is listed in `RegexandParsing/IntroductiontoRegexModule.py`

As your solution is updated, I suggest to do the following:
- Accommodate the code review comment.
- Delete that file `RegexandParsing/IntroductiontoRegexModule.py`.
- Update the `readme.md` to list `RegexandParsing/DetectFloatingPointNumbers.py`

Thank you.

File: RegexandParsing/DetectFloatingPointNumbers.py (hunk 1 above)
Comment (HighnessAtharva) [reply RegexandParsing/DetectFloatingPointNumbers.py]: Certainly. Done.

**Comment by HighnessAtharva** [comment]
- Regex Pattern improvised.
- Previous obsolete file deleted 
- Readme updated with DetectFloatingPointNumbers.py in place of IntroToRegex.py

Please review and merge.

**Review by arsho — APPROVED** [review]
Facilitated the change request smoothly. Thank you.

**Pull request merged**